geniza/corpus/management/commands/import_iiif_urls.py

```diff
@@ -37,6 +38,11 @@ def handle(self, *args, **options):
             for row in csvreader:
                 self.import_iiif_url(row)
 
+        logger.info(f"URLs added: {self.stats['added']}")
+        logger.info(f"URLs updated: {self.stats['updated']}")
+        logger.info(f"Fragments not found: {self.stats['not-found']}")
+        logger.info(f"Fragments skipped: {self.stats['skipped']}")
+
```

Comment:
FWIW, I prefer that we use django manage command output methods unless we have a compelling reason not to https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/

Reply:
Following the documentation isn't working for me (?) Getting the following error 
```
AttributeError: 'Command' object has no attribute 'stdout'
```
I simply tried replacing `logger.info` with `self.stdout.write`. If my mistake is obvious, let me know. Otherwise I'm just going to put this to the side.

Reply:
It's because you overrode the init method without calling super (sorry I didn't catch this before). I think it's not recommended to override init in manage commands, but you could check the django docs